src/app/components/Figure/Image/index.amp.jsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Img } from '.';
+
+const AmpImg = () => <Img as="amp-img" layout="responsive" />;
```

Comment:
Nice 👍 

EDIT: we should still make it possible to override `layout`, just default it to `responsive`

Reply:
Okay, I didn't realise that we'd need that for this initial version. If we're going to make it a prop, which other values do you think we'll need to use? `intrinsic` and `fixed`? https://www.ampproject.org/docs/design/responsive/control_layout#the-layout-attribute

I'll want to add a unit test to capture the variance in this.

Reply:
Ha - fair point! After looking at all the possible attributes we _could_ add - which feels like a premature optimisation - I'm now thinking we should keep it as hard-coded `layout="responsive"`.

EDIT - we don't even need to explicitly set the `layout`, as [AMP infers the layout behaviour from the width/height attributes](https://www.ampproject.org/docs/design/responsive/control_layout#what-if-the-layout-attribute-isn%E2%80%99t-specified?). So I'm now thinking we leave it out entirely, until it's needed!